### PR TITLE
#154079729 - Implement search logic in search component

### PR DESCRIPTION
**Pull request by peterpaints**
#### What does this PR do?
Implements Search Logic
#### Description of Task to be completed?
As an Admin, WHEN I visit the Incident search page THEN I should be able to search for Incidents.
#### Where should the reviewer start?
View file changes on PR.
#### How should this be manually tested?
- Ensure you're running in development mode by setting NODE_ENV=development in .env
- Open the dashboard and click on the search input on the navbar.
- Search for an incident, you should see relevant results as you type.
#### What are the relevant pivotal tracker stories?
#154079729
#### Screenshots (if appropriate)
screen shot 2018-02-27 at 2 35 01 pm
screen shot 2018-02-27 at 2 35 19 pm

### CODE REVIEW

**Comment by NaiRobley** [comment]
@peterpaints rebase from `staging` to sort the `package.json` and resolve the conflicts.

**Comment by peterpaints** [comment]
@NaiRobley I have rebased and fixed the conflicts

File: package.json [hunk 1]
```diff
@@ -43,7 +45,10 @@
         "react-router-dom": "^4.2.2",
         "react-router-redux": "^4.0.8",
         "redux": "^3.7.2",
-        "redux-thunk": "^2.2.0"
+        "redux-thunk": "^2.2.0",
+        "webpack": "^3.10.0",
+        "webpack-dev-middleware": "^1.12.0",
```
Comment (NaiRobley) [inline package.json]: Rebase and update this dependencies listing

Comment (peterpaints) [reply package.json]: @NaiRobley I have fixed them.

**Pull request merged**